### DISCUSSION TITLE
PT database

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#1990] [General]
I have a brain fade 😂, but where could I get a copy of Project Tracker database to install locally?
Regards

**Comment by rockfordlhotka** [comment]
I quit "shipping" the localdb database files because it was too painful to keep them in sync with different sql express and VS versions.

Instead, you can use the sql file to create the tables, etc.

https://github.com/MarimerLLC/csla/blob/master/Samples/ProjectTracker/ProjectTracker.DalEfCore/PTrackerData.sql

**Reply by Chicagoan2016** [reply]
Thank you Sir!
